### ISSUE TITLE
missing implementation of itk::GPUDenseFiniteDifferenceImageFilterKernel::GetOpenCLSource()

### ISSUE THREAD

**Issue by chacha21** [InsightSoftwareConsortium/ITK#1889]
ITK 5.1

I am using itkGPUPDEDeformableRegistrationFilter, but the link stage fails because of missing itk::GPUDenseFiniteDifferenceImageFilterKernel::GetOpenCLSource()

adding the following in any unit of compilation will fix it.
```
#include <itkGPUDenseFiniteDifferenceImageFilter.h>
namespace itk {
const char* GPUDenseFiniteDifferenceImageFilterKernel::GetOpenCLSource() {return 0;}
}
```

**Comment by dzenanz** [comment]
I think this problem was resolved after 5.1 was tagged. Can you try master?

**Comment by chacha21** [comment]
Just tried, it is fixed in current master
